src/exporters/era5_land.py

```diff
@@ -0,0 +1,178 @@
+                val = self._check_iterable(val, key)
+                processed_selection_request[key] = [self._correct_input(x, key) for x in val]
+        return processed_selection_request
+
+    def export(self,
+               variable: str,
+               show_api_request: bool = True,
+               selection_request: Optional[Dict] = None,
+               break_up: Optional[str] = 'yearly',
+               n_parallel_requests: int = 1) -> List[Path]:
+        """ Export functionality to prepare the API request and to send it to
+        the cdsapi.client() object.
+
+        Arguments:
+        ---------
+        variable: str
+            The variable to be exported
+        show_api_request: bool = True
+            Whether to print the selection dictionary before making the API request
+        selection_request: Optional[Dict], default = None
+            Selection request arguments to be merged with the defaults. If both a key is
+            defined in both the selection_request and the defaults, the value in the
+            selection_request takes precedence.
+        break_up: str: {'yearly', 'monthly'}, default = 'yearly'
+            The best way to download the data is by relatively large calls to the CDS
+            API. If specified, the calls will be broken up by {'yearly', 'monthly'}
+        parallel: bool, default = True
+            Whether to download data in parallel
+        n_parallel_requests:
+            How many parallel requests to the CDSAPI to make
+
+        Returns:
+        -------
+        output_files: List of pathlib.Paths
+            paths to the downloaded data
+        """
+        dataset = self.dataset
+        granularity = self.granularity
+
+        if break_up is not None:
+            assert break_up in ['yearly', 'monthly'], f'Only 2 valid ways to ' \
+                'break up requests: {"yearly", "monthly"}'
+
+        # create the default template for the selection request
+        processed_selection_request = self.create_selection_request(variable,
+                                                                    selection_request,
+                                                                    granularity)
+
+        if n_parallel_requests < 1: n_parallel_requests = 1
+
+        # break up by month
+        if break_up == 'monthly':
+            if n_parallel_requests > 1:  # Run in parallel
+                p = multiprocessing.Pool(int(n_parallel_requests))
+
+            output_paths = []
+            for year, month in itertools.product(processed_selection_request['year'],
+                                                 processed_selection_request['month']):
+                updated_request = processed_selection_request.copy()
+                updated_request['year'] = [year]
+                updated_request['month'] = [month]
+
+                if n_parallel_requests > 1:  # Run in parallel
```

Comment:
Nit: a lot of this code is line-for-line copied in the monthly and yearly case - is it worth adding some sort of `_broken_export` function?

Reply:
Okay i think i put that into practice!
thanks